package.json

```diff
@@ -1,8 +1,10 @@
 {
   "license": "MIT",
   "devDependencies": {
-    "parcel-bundler": "^1.12.5",
-    "purescript": "^0.14.3",
+    "esbuild": "^0.14.49",
+    "parcel": "^2.6.0",
+    "process": "^0.11.10",
+    "purescript": "^0.15.3",
     "spago": "^0.20.3"
```

Comment:
These dependencies need to be updated:
- `spago`: 0.20.9 - running any of the recipes will fail because this version of Spago runs code as thought it were CJS modules.
- `purescript`: we should bump this to `0.15.4` now.

Reply:
`^` is a version range and no `package-lock.json` is checked in. Therefore any of the listed updates are included or will be installed.

If specific version should be used, I would rather switch to fixed versions and/or checking in the `package-lock.json`.

Reply:
> Therefore any of the listed updates are included or will be installed.

You're right. Hmm.... When I ran `make installDeps`, it didn't update my dependencies. When I `rm -rf node_modules && make installDeps` then it updated them.

I don't think we should use fixed versions because then we'd have to update this repo every time a new release was made, so the version ranges make sense.

Reply:
One explanation could be that you created a `package-lock.json` with a previous installation. The file is git-ignored, but still exists locally, and would be respected. I think npm wouldn't just replace installed versions if they match the `package-lock.json` if there are newer versions available.

Reply:
Ah... Yeah, I think that's what happened.